src/quiz/useQuiz.js

```diff
@@ -0,0 +1,8 @@
+import { useContext } from 'react';
+import QuizContext from './QuizContextProvider';
+
+const useQuiz = () => {
```

Comment:
Awesome use of a context provider!

Reply:
Thanks! It was Kat's idea (and she implemented the authContext, which I used as a template)!